codersdk/client.go

```diff
@@ -15,8 +15,13 @@ import (
 	"nhooyr.io/websocket"
 )
 
-// SessionTokenKey represents the name of the cookie or query parameter the API key is stored in.
-const SessionTokenKey = "session_token"
+const (
+	// SessionTokenKey represents the name of the cookie or query parameter the API key is stored in.
+	SessionTokenKey = "session_token"
+
+	OAuth2StateKey    = "oauth_state"
+	OAuth2RedirectKey = "oauth_redirect"
```

Comment:
Could we add a note here about updating the strip cookies function if new ones are added? Or potentially putting these in an array here which is accessed by the strip function. If new sensitive cookies are added, could be easy to miss otherwise.

It would also be nice if our cookies had a prefix (to avoid above problem), but there may be good reasons not to have it.

Reply:
Indeed sir! Great idea.

Reply:
Hmm... that's a good point with the prefix.

Reply:
Hmm... that's a good point with the prefix.